.github/workflows/docs-deploy.yaml

```diff
@@ -5,23 +5,26 @@ on:
     branches:
       - main
     paths:
-      - 'docs/**'
+      - "docs/**"
   push:
     branches:
       - main
     paths:
-      - 'docs/**'
+      - "docs/**"
   workflow_dispatch:
 
 jobs:
   build:
     name: Build Docusaurus
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

Comment:
📣 Permissions for the `build` step are proving alright with just `read`! This is interesting to me, but makes sense in the context of uploading artifacts to action outputs perhaps?

📚 https://github.com/slackapi/java-slack-sdk/actions/runs/15126115805?pr=1474